style.css

```diff
@@ -247,7 +248,8 @@ div#frams {
 }
 
 .but1:hover {
-  background-color: #e7af87;
+  background: #fff5e1;
+  border: 1px solid #f55800;
 }
```

Comment:
- [Optional ] On hover the text in the button disappeared because the color of both the text and the background is white, kindly change the text color to match the Figma design.